libs/librrgraph/src/base/alloc_and_load_rr_indexed_data.h

```diff
@@ -0,0 +1,12 @@
+#ifndef ALLOC_AND_LOAD_RR_INDEXED_DATA_H
```

Comment:
This is a utility function. Consider to move them to a separated directory ``libs/librrgraph/src/utils``